tests/validation/tests/v3_api/common.py

```diff
@@ -1022,6 +1030,19 @@ def retry_dig(host, pod, expected, retry_count=3):
             time.sleep(3)
     pytest.fail(f"failed to get the expected number of dns hosts from dig")
 
+def retry_cmd(host, pod, cmd, expected, retry_count=5):
+    for i in range (0,retry_count):
+        if HARDENED_CLUSTER:
```

Comment:
I feel like without this if/else statement, the 2 functions you've added are the same and could be condensed into one. (not necessary, but it looks like it could be done once you rm this if/else)